README.md

```diff
@@ -1,36 +1,30 @@
-![Stage: Stable Release](https://img.shields.io/badge/stage-stable-green)
-![Release](https://img.shields.io/github/v/release/auth0/auth0-angular)
-[![CircleCI](https://img.shields.io/circleci/build/github/auth0/auth0-angular)](https://circleci.com/gh/auth0/auth0-angular)
-[![Codecov](https://img.shields.io/codecov/c/github/auth0/auth0-angular)](https://codecov.io/gh/auth0/auth0-angular)
-![Downloads](https://img.shields.io/npm/dw/@auth0/auth0-angular)
-
 # Auth0 Angular SDK
 
 A library for integrating [Auth0](https://auth0.com) into an Angular application.
 
-## Table of Contents
+![Release](https://img.shields.io/github/v/release/auth0/auth0-angular)
```

Comment:
One question here, do we want `Release` to show the the latest release from a time perspective (i.e. including betas, previous majors that have been updated) or from a "latest GA release" perspective? I think using GitHub for the source gives us the former whereas npm/other package managers would give the latter

 For example [spa-js](https://github.com/auth0/auth0-spa-js/tree/master) currently shows `2.0.0-beta.1` which while technically correct doesn't feel like the version we want shown (1.22.5 imo)

Reply:
That's a good point. Let me update any of the open PR's I have that use this, thanks!